packages/onboarding/src/setup-tailored-site-after-creation.ts

```diff
@@ -36,7 +36,7 @@ interface SetupOnboardingSiteOptions {
 }
 
 export function setupSiteAfterCreation( { siteId, flowName }: SetupOnboardingSiteOptions ) {
-	const { resetOnboardStore } = dispatch( ONBOARD_STORE );
+	// const { resetOnboardStore } = dispatch( ONBOARD_STORE );
```

Comment:
Instead of committing commented out code, is removing it OK?

Reply:
Sure.
My concern with the reset in this file is breaking other flows.
I've removed the call from this branch and added the reset call in the tailored flow file, but I haven't tested yet how it might affect other existing flows.

Reply:
@Automattic/serenity  @mreishus hi! 

I've been noticing bugs and odd behaviours (values persisting) in onboarding flows or wrong theme appearing to the site cross different flows, and I suspect it's because the reseting onboard store has been disabled here.

Was there ever a follow up to above discussion and re-enabling the reset?